src/marketo/marketo.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Ceros Marketo Plugin
+ * @version 0.1.0
+ * @support support@ceros.com
+ *
+ * This plugin is licensed under the MIT license. A copy of this license and
+ * the accompanying source code is available at https://github.com/ceros/ceros-plugins
+ */
+(function() {
+    if (typeof(CerosSDK) === "undefined") {
+        var sdkScript = document.createElement('script');
+        sdkScript.type = "text/javascript";
+        sdkScript.async = true;
+        sdkScript.onload = activateMarketoTracking;
+        sdkScript.src = "//sdk.ceros.com/standalone-player-sdk-v3.js";
+
+        document.getElementsByTagName('head')[0].appendChild(sdkScript);
+    }
+    else {
+        activateMarketoTracking();
+    }
+
+
+    function activateMarketoTracking() {
+        var pluginScriptTag = document.getElementById("ceros-marketo-plugin");
+        var accountId = pluginScriptTag.getAttribute("accountId");
+
+        if (!accountId) {
+            console.error("Account ID is required for the Ceros Munchkin plugin.");
+        }
+
+        var didInit;
+        function initMunchkin() {
+            if(!didInit) {
+                didInit = true;
+                Munchkin.init(accountId);
+            }
+        }
+
+        //load the Marketo script then initialize
+        var marketoScript = document.createElement('script');
+        marketoScript.type = "text/javascript";
+        marketoScript.async = true;
+        marketoScript.src = '//munchkin.marketo.net/munchkin.js';
+        marketoScript.onreadystatechange = function() {
+            if (this.readyState == 'complete' || this.readyState == 'loaded') {
```

Comment:
I mostly have it there because it was in the marketo examples, and its not on line 14 because it wasnt in the eloqua plugin i stole it form.

if we have it in on lace it should be in both for sure, but don't we care that the marketo script has loaded before firing the init?